.github/workflows/release.yml

```diff
@@ -116,6 +116,7 @@ jobs:
       run: |
           docker login --username "${DOCKERIO_USERNAME}" --password "${DOCKERIO_PASSWORD}"
           ./base-images/build.sh
+          DOCKER_IMAGE_TAG=latest ./base-images/build.sh
```

Comment:
What will happen if we release a new version? Will we still use latest as the tag?

Reply:
yes, latest should always track on the newest release